USE-CASES/digital-twin-use-case.md

```diff
@@ -0,0 +1,65 @@
+## Title: Digital Twin
+
+### Submitter(s): 
+
+Qing An
+
+### Reviewer(s):
+
+
+### Tracker Issue ID:
+
+
+
+### Category:
+
+horizontal
+
+### Class: 
+
+
+
+### Status: 
+
+
+
+### Target Users
+Digital twin involves managing a physical device or a group of connected physical devices which needs to be virtually represented, and whose data needs to be understood.
+Stakeholders include:
+- device owners: need to make data from devices available to digital twin system.
+- device user: users of the digital twin system are indirectly using the devices by accessing their data generated by the devices and sent to the digital twin, and also, by sending commands to the digital twin and corresponding actions can be automatically invoked on the device.
+- cloud or edge provider: the digital twin system may be hosted in the cloud or in the edge.
+
+### Motivation:
+
+A digital twin is a virtual and digital representation, located in the cloud or in the edge, of a real-world entity or system that mirrors a unique physical object, or a group of connected physical devices. 
+
+Simply by describing a single device’s functionalities is not enough to support the accurate virtual representation in the cloud or edge. To accurately simulate the physical entity or system, the real-time status of device, the relation and rules among a collection of connected devices need be standardized.
```

Comment:
"Realtime" causes some discussions - people understand hard realtime - consider "time sensitive updates"

Simulation to virtually represent devices in the cloud - simulate existing status and relationships
Describing a simulation model in the TD would be useful, what-if rules and relationships.

We have the tools for modeling the relationship (links container), this can express any kind of relationship.

Next step: provide more details, real code